org.eclipse.winery.bpmn4tosca.converter.tobpel/.gitignore

```diff
@@ -22,6 +22,7 @@ build/
 *.jar
 *.war
 *.ear
+*.zip
```

Comment:
Is this really necessary?  Doesn't the test now create the files in a temporary folder?